src/endpoints/transfers_recipients.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+module.exports = {
+  /*
+    Create Transfer Recipient
+    @params: type, name, account_number, bank_code, currency, description, authorization_code
+  */
+  createTransferRecipient: {
+    method: 'POST',
+    path: '/transferrecipient',
+    send_json: true,
+    params: { type: String, name$: String, account_number$: String, bank_code$: String, currency: String, description: String, authorization_code: String },
```

Comment:
For this line, the _type_ param is required according to the [paystack reference section](https://developers.paystack.co/reference#create-transfer-recipient) and so should have the *$* sign at the end of it (the *$* sign at the end marks a given param as required).

Reply:
ah right, got it. Will update.